githublabels.py

```diff
@@ -10,12 +10,14 @@ def get_dpg_pog(): return []
   "rejected": "e11d21",
   "info": "0000ff",
   "doc": "257fdb",
+  "manually-approved": "defb04",
 }
 
 LABEL_TYPES = {
   "pending":  LABEL_COLORS["pending"],
   "approved": LABEL_COLORS["approved"],
   "rejected": LABEL_COLORS["rejected"],
+  "manually-approved": LABEL_COLORS["manually-approved"],
```

Comment:
use `info` instead of of new color

Reply:
@iarspider , label will not be called `manually-approved` and we donto want it one per cmssw category, so please remove it fro here

Reply:
labels should be added in `COMMON_LABELS ` dict and we should have one label for each of these `manual-override, ib-failure, external-failure` e.g `tests-manual-override`, `tests-ib-failure`. 

I would suggest to move the `TEST_IGNORE_REASON`  from process_pr.py to here and import it in process_pr

Reply:
done